<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conductor</class>
 <widget class="QWidget" name="Conductor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lEstSalida">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ESTACION SALIDA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lIDConductor">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID CONDUCTOR</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditHoraLlegada">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>118</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lNPasajeros">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>N° PASAJEROS</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditFecha">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditHoraSalida">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>118</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lNUnidad">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>N° UNIDAD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEEstacionSalida">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lEstLLegada">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ESTACION LLEGADA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lHoraSalida">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HORA SALIDA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lHoraLLegada">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HORA LLEGADA</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEEstLlegada">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lFecha">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FECHA</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineENPasajeros">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="titleConductores">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CONDUCTORES</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineENUnidad">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEIDConductor">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
